viite-UI/manual/loppukayttajaohje.md

```diff
@@ -51,7 +51,7 @@ Sovelluksen toimiessa virheellisesti (esim. kaikki aineistot eiv&auml;t lataudu
 
 - Lataa sivu uudelleen n&auml;pp&auml;imist&ouml;n F5-painikkeella.
 - Tarkista, ett&auml; selaimestasi on k&auml;yt&ouml;ss&auml; ajan tasalla oleva versio ja selaimesi on Mozilla Firefox tai Chrome
-- Jos edell&auml; olevat eiv&auml;t korjaa ongelmaa, ota yhteytt&auml; janne.grekula@karttakeskus.fi
+- Jos edell&auml; olevat eiv&auml;t korjaa ongelmaa, ota yhteytt&auml; janne.grekula@cgi.com
```

Comment:
This email address should be changed to some other than personal email address. Maybe operator's address might be good here?

Reply:
I second that. Good recommendation.